sda-commons-server-morphia/src/main/java/org/sdase/commons/server/morphia/internal/TracingCommandListener.java

```diff
@@ -0,0 +1,31 @@
+package org.sdase.commons.server.morphia.internal;
+
+import com.mongodb.event.CommandFailedEvent;
+import com.mongodb.event.CommandListener;
+import com.mongodb.event.CommandStartedEvent;
+import com.mongodb.event.CommandSucceededEvent;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.mongo.v3_1.MongoTelemetry;
+
+public class TracingCommandListener implements CommandListener {
+  private final CommandListener delegate;
+
+  public TracingCommandListener(OpenTelemetry openTelemetry) {
+    delegate = MongoTelemetry.builder(openTelemetry).build().newCommandListener();
+  }
+
+  @Override
+  public void commandStarted(CommandStartedEvent event) {
+    delegate.commandStarted(event);
+  }
+
+  @Override
+  public void commandSucceeded(CommandSucceededEvent event) {
+    delegate.commandSucceeded(event);
+  }
+
+  @Override
+  public void commandFailed(CommandFailedEvent event) {
+    delegate.commandFailed(event);
+  }
```

Comment:
Do we need to sanitize the commands?

Reply:
This is handled internally, there is a [test](https://github.com/SDA-SE/sda-dropwizard-commons/pull/1791/files#diff-5358a69c6bee4077ae216ce02b9f229c0a05ce2906a25a5035addf2d6175b64dL79-L81) for that .